### PR TITLE
Navigation with javascript

### PR DESCRIPTION
**Pull request by sjdev2212**
In this project I: 
Created a Nav-bar.
Created a contact section.
Created a footer.
Styled the new sections.
Implemented navigation links using javascript.

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @sj1978, 

Good job so far! Git-flow used :+1: Navigation bar create :100: Three main tab pages created :1st_place_medal: 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @sj1978 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// eslint-disable-next-line max-classes-per-file
+/* eslint-disable no-undef */
+/* eslint-disable max-classes-per-file */
 class BookShelf {
   constructor(title, author, id) {
     // eslint-disable-next-line no-unused-expressions
```
[the remark this answers is not in the document]
Comment (sj1978) [reply index.js]: To whom does my review 
I'm sending my last submit.
I could fix my eslint problem regarding classes but unfortunately, I couldn't fix the eslint issue with luxon.
I have been trying different things for hours with no result.
So I have just one linter disabled . 
Hope it is ok to pass.
Thanks in advance.

[the remark this answers is not in the document]
Comment (henatan99) [reply index.js]: The effort you put to fix linters issues you could fix is appreciated. Now we can make the rest optional. :clap:

**Review by henatan99 — APPROVED** [review]
Hi @sj1978 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**